RepositoryCompiler/CodeModel/CodeParsers/CSharp/CSharpCaDETMemberBuilder.cs

```diff
@@ -112,6 +113,14 @@ private List<CaDETParameter> GetMethodParams()
             return memberParams;
         }
 
+        private List<string> GetMethodVariableNames()
+        {
+            var variables = _cSharpMember.DescendantNodes().OfType<VariableDeclaratorSyntax>().ToList();
+            List<string> variableNames = new List<string>();
```

Comment:
Lines 119-121 can be reduced to "return variables.Select(v => v.Identifier.ToString())"